source/configuration/sharding.txt

```diff
@@ -0,0 +1,161 @@
+.. _mongoid-sharding-configuration:
+
+======================
+Sharding Configuration
+======================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: ruby framework, odm, code example
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Sharding is a way to distribute your data across multiple machines. MongoDB
+uses sharding to support deployments with large data sets and high
+throughput operations. In this guide, you can learn how to configure sharding in
+your {+odm+} application.
+
+Declare Shard keys
+------------------
+
+MongoDB uses shard keys to distribute a collection's documents across
+shards. A shard key consists of one or more fields in a document. In your
```

Comment:
S: i like the manual's explanation of shard keys. might be worth incorporating

https://www.mongodb.com/docs/manual/core/sharding-shard-key/

```suggestion
MongoDB uses shard keys to distribute a collection's documents across
shards. A shard key is an indexed field, or multiple fields covered by a
compound index, that determines the distribution of the collection's
documents among the cluster's shards. In your
```

Reply:
I stole mine from a different page in the manual, but agreed this one is better